Eatery Blue/UI/HomeScreen/HomeViewController.swift

```diff
@@ -115,8 +120,8 @@ class HomeViewController: UIViewController {
         updateScrollViewContentInset()
     }
 
-    func scrollToTop() {
-        tableView.scrollToRow(at: IndexPath(row: 0, section: 0), at: .top, animated: true)
+    func scrollToTop(animated: Bool) {
+        tableView.scrollToRow(at: IndexPath(row: 0, section: 0), at: .top, animated: animated)
     }
```

Comment:
Can you factor this out to a UITableView extension?

Reply:
I don't think we should because there is a separate `scrollToTop()` function in `HomeSearchContentViewController` for its own UITableView that is defined differently.

Reply:
Is there a way you can make a general `scrollToTop` function based on the arguments that you passed in?

Reply:
We only have 1 table view and 2 `scrollToTop` functions which are 1 line each so I don't think that's necessary